exasol/nb_connector/connections.py

```diff
@@ -154,7 +83,8 @@ def get_udf_bucket_path(conf: Secrets) -> str:
     Builds the path of the BucketFS bucket specified in the configuration,
     as it's seen in the udf's file system.
     """
-    return f"/buckets/{conf.get(CKey.bfs_service)}/{conf.get(CKey.bfs_bucket)}"
+    bucket = open_bucketfs_connection(conf)
+    return bucket.udf_path
```

Comment:
Wouldn't the return type "Pathlike" would fit better for this function, instead of 'str'?
Sorry if this question is not related to the changes in the PR.

Reply:
Maybe, but then I would break compatibility and will have to change few notebooks in the AI-Lab.